effectiveOfficeBackend/src/main/kotlin/office/effective/features/user/repository/UserRepository.kt

```diff
@@ -87,5 +87,32 @@ class UserRepository {
         return UserTagModel(tag.id, tag.name)
     }
 
+    fun updateUser(model: UserModel): UserModel {
+        model.id ?: throw UserNotFoundException("No id in the model")
+        if (!existsById(model.id!!)) {
+            throw UserNotFoundException("User ${model.fullName} with id:${model.id} does not exists")
+        }
+        var ent = db.users.find { it.id eq model.id!! }
+        ent?.tag = model.tag
+        ent?.fullName = model.fullName
+        ent?.active = model.active
+        ent?.avatarURL = model.avatarURL
+        ent?.role = model.role
+        ent?.flushChanges()
+        //todo integrations
```

Comment:
We agreed that if we write a TODO, we mention who it applies to and leave a link to the board with the task.

Note: TODO(Egor Parkhomenko **`link`**) ...